base/download.jl

```diff
@@ -4,7 +4,7 @@
 
 if Sys.iswindows()
     function download(url::AbstractString, filename::AbstractString)
-        ps = "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe"
+        ps = joinpath(get(ENV, "SYSTEMROOT", "C:\\Windows"), "System32\\WindowsPowerShell\\v1.0\\powershell.exe")
```

Comment:
@vtjnash  are you ok with removing the fallback to `C:\\Windows` , I think it's superfluous 

You would have much bigger problems  than  anything to do with Julia if someone fudged this env variable

Reply:
Conversely, it there any reason not to keep it?

Reply:
Well if SYSTEMROOT didn't' work to find powershell, there's no way "C:\\Windows" would either, so I think it's superfluous (not a big deal, but just saying).